source/develop/release-management/features/storage/incremental-backup.md

```diff
@@ -228,8 +228,9 @@ Response:
 
 #### Starting full backup
 
-Start full backup. The response includes 'to_checkpoint_id' which
-is created on the disks. It can be used in the next incremental backup.
+Start full backup. The response will not include 'to_checkpoint_id' which
+should be used as the 'from_checkpoint_id' in the next incremental backup,
+only when the backup phase will be "ready".
```

Comment:
This not clear.

Should be something like:

Start full backup. The response phase indicates that the backup is "initializing". You need to poll the backup until the phase is "ready". Once the backup is ready the response will include "to_checkpoint_id" ...

Reply:
rephrased